hvplot/plotting/core.py

```diff
@@ -1295,6 +1296,20 @@ def hist(self, y=None, by=None, **kwds):
             df['two'] = df['one'] + np.random.randint(1, 7, 6000)
             df.hvplot.hist(bins=12, alpha=0.5, color=["lightgreen", "pink"])
 
+        If you want to show the distribution of the values of a categorical column,
+        you can use `value_counts` and `bar` as shown below
+
+        .. code-block::
+
+            import hvplot.pandas
+            import pandas as pd
+
+            data = pd.DataFrame({
+                "library": ["bokeh", "plotly", "matplotlib", "bokeh", "matplotlib", "matplotlib"]
+            })
+
+            data["library"].value_counts().hvplot.bar(invert=True, flip_yaxis=True)
```

Comment:
It seems using `invert` and `flip_yaxis` is confusing here, they seem to be required while as far as I understand they're just there for stylistic reasons.

Reply:
What do you mean by *seams confusing* ?

For context I put them in because otherwise you will not get readable and useful plots.

Reply:
This is the plot I get without `invert=True, flip_yaxis=True`, which doesn't look that bad and is closer to what I expect from a *distribution* plot, which is the intent of this added example.

![image](https://user-images.githubusercontent.com/35924738/225477397-e1010909-93d9-4fac-8977-3be979105372.png)

> I also think its really nice to demonstrate a lot of the parameters

Totally agree with that, but not in the docstrings, as there are a very large number of examples we could put in there, and without any visual reference they are not going to help "potential buyers" much.

```suggestion
            data["library"].value_counts().hvplot.bar()
```